src/Command/SeedCommand.php

```diff
@@ -136,10 +157,54 @@ protected function executeSeeds(Arguments $args, ConsoleIo $io): ?int
 
         $start = microtime(true);
         if (!$seeds) {
+            // Get all available seeds and ask for confirmation
+            try {
+                $availableSeeds = $manager->getSeeds();
+            } catch (Exception $e) {
+                $io->error('Failed to load seeds: ' . $e->getMessage());
+
+                return static::CODE_ERROR;
+            }
```

Comment:
I think that defaulting to running all seeds is a risky default, that should be an error instead.  If there is a need for a 'run all' mode we could add a `--all` option.